app_dart/test/request_handlers/refresh_github_commits_test.dart

```diff
@@ -172,6 +171,15 @@ void main() {
     test('skips commits for which transaction commit fails', () async {
       githubCommits = <String>['1', '2', '3'];
       config.flutterBranchesValue = <String>['master'];
+
+      /// Pre-insert one commit first, otherwise it will insert only one
+      /// commit for a new branch.
+      const List<String> dbCommits = <String>['4'];
+      for (String sha in dbCommits) {
+        final Commit commit = shaToCommit(sha, 'master');
+        db.values[commit.key] = commit;
+      }
+
```

Comment:
1. The comment doesn't address why a pre-insert is needed. The reason is that this test is simulating an existing branch, which must already have at least one processed commit in the datastore.
2. The existing processed commit should have smaller timestamp than `githubCommits`.
3. Since there's only one element, there's no need for a loop.

Reply:
1. updated.
2. updated existing commit to have smaller timestamp.
3. removed loop.